documentation/src/main/asciidoc/topics/ref_rest_counters.adoc

```diff
@@ -123,9 +128,21 @@ POST /rest/v2/counters/{counterName}?action=decrement
 `STRONG` counters return `200 (OK)` and the current value after each operation.
 ====
 
+[id='rest_server_counters_set']
+= Performing GET and SET Operations on Strong Counters
```

Comment:
@dvagnero what about?
```suggestion
= Performing "get and set" atomic operations on Strong Counters
```
"getAndSet" is a single operation :)

Reply:
Ah okay I'd go with ``getAndSet``
```suggestion
= Performing `getAndSet` atomic operations on Strong Counters
```

Reply:
Should i make this change ? @dvagnero

Reply:
Yes go ahead. I think it's a good title :)